src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,34 @@
+import "./Introduce.css";
+
+const Introduce = () => {
+  return (
+    <main>
+      <div className="introduce">
+        <img
+          className="myprofile"
+          src="images/myprofile.JPG"
+          alt="프로필사진"
+        />
+        <p className="name">장건웅</p>
+        <p className="YearTh">46기</p>
```

Comment:
```suggestion
        <p className="yearTh">46기</p>
```
첫 글자가 대문자인 컨벤션은 컴포넌트의 이름일때 적용하는 컨벤션입니다.
className일때는 다른 case로 적용해주세요!